examples/trials/mnist-pytorch/config_detailed.yml

```diff
@@ -0,0 +1,42 @@
+# This example shows more configurable fields comparing to the minimal "config.yml"
+# You can use "nnictl create --config config_detailed.yml" to launch this experiment.
+# If you see an error message saying "port 8080 is used", use "nnictl stop --all" to stop previous experiment.
```

Comment:
-> use "nnictl stop --port 8080" to stop that experiment, or use "nnictl stop --all" to stop all the previous experiments.

Reply:
I prefer not to provide that much details in example. At this point the user does not need to know how to manage multiple experiments.
I'm afraid that "--port 8080" might threaten newbie users.